src/common/select-objects.js

```diff
@@ -32,6 +32,15 @@ import {
 } from 'xo'
 
 // ===================================================================
+//
+// GenericSelect usage:
+//
+// One subclass must inherit from GenericSelect and implement
+// _computeOptions (props).
+// GenericSelect must receive the xoObjects prop. It's a hash with
+// the XO objects that can be selected. The keys are the objects ids.
```

Comment:
Give an example of creating a select relying on the store and another select relying on a subscription.